pkg/components/microservice.go

```diff
@@ -46,6 +46,7 @@ type microserviceImpl struct {
 	builtDeployment *appsv1.Deployment
 	builtService    *corev1.Service
 	builtConfig     *corev1.ConfigMap
+	tolerations     []corev1.Toleration
```

Comment:
It looks like you don't need it in this struct

Reply:
fixed